settings/opendal.yml

```diff
@@ -0,0 +1,5 @@
+# Apache OpenDAL settings file
```

Comment:
This file shouldn't probably be part of the `contribcard` repository. We left the Kubernetes one as an example until we document it and move it to `docs/config` (similarly to [what we do in the landscape2 repo](https://github.com/cncf/landscape2/tree/main/docs/config)). But projects settings files should probably be hosted somewhere else (actually, Kubernetes' one is hosted at [cncf/contribcard-sites](https://github.com/cncf/contribcard-sites), along with the workflows that build and deploy the site).

Reply:
OK. Let me remove this.

Reply:
Removed.